pkg/plugin/plugin.go

```diff
@@ -109,9 +114,50 @@ func loadNetConf(bytes []byte) (*netConf, error) {
 		return nil, fmt.Errorf("failed to load netconf: %v", err)
 	}
 
+	// default config file dir paths
```

Comment:
Can we move this whole new block loading the file to a separate function? And then have one top level function which would merge `loadNetConf` and `loadFlatNetConf`? Would be easier to follow IMO.

Reply:
Good Point! done it now.